Sources/SwiftDocC/Infrastructure/NodeURLGenerator.swift

```diff
@@ -92,7 +94,8 @@ public struct NodeURLGenerator {
                         isDirectory: false
                     )
                     .path
-            case .technology(let technologyName):
+            case .technology(let technologyName),
+                 .tutorialTableOfContents(let technologyName):
```

Comment:
```suggestion
                 .tutorialTableOfContents(let tutorialName):
```

Reply:
I originally left it this way because both cases need to bind the same name and a tutorial table of contents page is conceptually about a technology. I renamed it to `name` in https://github.com/swiftlang/swift-docc/pull/1039/commits/5b43b7c13da335751b0ff3c024db7674ec8dc7b9